src/I18n/DateFormatTrait.php

```diff
@@ -101,7 +101,7 @@ protected function _formatObject(
             static::$formatters[$key] = $formatter;
         }
 
-        return (string)static::$formatters[$key]->format($date->format('U'));
+        return (string)static::$formatters[$key]->format($date->format('U.u'));
```

Comment:
`IntlDateFormatter::format()` also accepts `DateTimeInterface` instances which `$date` already is. So I wonder why is it not directly used instead of converting to string.

Reply:
I did not think about it, but your solution is in my opinion better

Reply:
I noticed this before as well. I must not have changed it after the Chronos 3 upgrade.